interface/patient_file/summary/stats_full.php

```diff
@@ -202,220 +181,252 @@ function newEncounter() {
-                    <table class="table">
+                <?php foreach ($ISSUE_TYPES as $t => $focustitles) : ?>
+                    <?php
+                    if (!AclMain::aclCheckIssue($t)) {
+                        continue;
+                    }
+
+                    if ($category && ($t !== $category)) {
+                        continue;
+                    }
+
+                    $btnDelete = AclMain::aclCheckCore('admin', 'super');
+                    $canSelect = $btnDelete;
+                    $btnAdd = false;
+                    if (AclMain::aclCheckIssue($t, '', ['write', 'addonly'])) {
+                        if (in_array($t, ['allergy', 'medications']) && $GLOBALS['erx_enable']) {
+                            $btnAdd = [
+                                'href' => '../../eRx.php?page=medentry',
+                                'onclick' => 'top.restoreSession()',
+                            ];
+                        } else {
+                            $btnAdd = [
+                                'href' => '#',
+                                'onclick' => 'dopclick(0, ' . attr_js($t) . ')'
+                            ];
+                        }
+                    }
+
+                    $condition = ($GLOBALS['erx_enable'] && $GLOBALS['erx_medication_display'] && $focustype == 'medication') ? "AND erx_uploaded != '1'" :  '';
+                    $pres = sqlStatement("SELECT * FROM lists WHERE pid = ? AND type = ? $condition ORDER BY begdate", [$pid, $t]);
+                    $noIssues = false;
+                    $nothingRecorded = false;
+
+                    // if no issues (will place a 'None' text vs. toggle algorithm here)
+                    if (sqlNumRows($pres) < 1) {
+                        if (getListTouch($pid, $focustype)) {
+                            // Data entry has happened to this type, so can display an explicit None.
+                            $noIssues = true;
+                        } else {
+                            // Data entry has not happened to this type, so can show the none selection option.
+                            $nothingRecorded = true;
+                        }
+                    }
+
+                    ?>
+                    <div class="bg-light w-100 p-3 d-flex sticky-top justify-content-between align-items-center">
+                        <div class="d-flex align-items-center">
+                            <?php if (!($noIssues || $nothingRecorded)) : ?>
+                                <input type="checkbox" class="selection-check mr-1" onclick="headerSelectionChanged(this, <?php echo attr_js($t);?>);"/>
+                                <button type="button" class="btn btn-text px-2" data-issue-type="<?php echo xla($t); ?>" data-action="toggle" data-expanded="false" aria-label="<?php echo xla("Expand or collapse all items in section"); ?>"><span class="fa fa-fw fa-expand" aria-hidden="true"></span></button>
```

Comment:
would above break in the "dummy" language. For example, if the data-issue-type was "dummy" for all these buttons would the ui break?

Reply:
Will move it to just an `attr()` call instead